README.md

```diff
@@ -8,40 +8,38 @@ Robo4J provides an easy way of getting started with building custom hardware and
 * [Robo4j.io][] provides a library of configurable units that allows hardware to be enabled and configured through XML
 * [Robo4j.io][] provides a threading model controlled by annotations
 
-The current [Robo4j.io][] version is 0.5
+The current [Robo4j.io][] version is 0.6-SNAPSHOT
 
 <a href="https://foojay.io/today/works-with-openjdk"><img align="left" src="https://github.com/foojayio/badges/raw/main/works_with_openjdk/Works-with-OpenJDK.png" width="100"></a>
 <br><br><br>
 
 ## Requirements
-[Git][], [Gradle][], [Maven][], [OpenJDK 11][]
+[Git][], [Maven][], [OpenJDK 21][]
 
-If you are looking for a JDK 11 ARM hard float build for Raspbian, we recommend looking into [Liberica JDK][] or [Azul Zulu Embedded][].
+If you are looking for a JDK 21 ARM hard float build for Raspbian, we recommend looking into [Liberica JDK][] or [Azul Zulu Embedded][].
```

Comment:
`[Liberica](https://bell-sw.com/pages/downloads/) or [Azul Zulu](https://www.azul.com/downloads/)`

Not Zulu Embedded please